ROADMAP.rst

```diff
@@ -48,13 +48,13 @@ This realease is to evaluate the features and functionality.
 - [ ] Key(s) Rotation (`Issue #23 <https://github.com/vmware/repository-service-tuf/issues/23>`_)
 - [ ] Option to Disable the API Authentication/Authorization (`Issue #41 <https://github.com/vmware/repository-service-tuf/issues/41>`_)
 
-(*Priority order*)
```

Comment:
Given that board doesn't preserve order do we want to remove the mention of the `Priority order` from here?

Reply:
At the current stage,  all priority features have tasks already signed. It may not be necessary at the moment.